pkg/controller/cache/cachesubnetgroup/controller_test.go

```diff
@@ -270,7 +270,7 @@ func TestObserve(t *testing.T) {
 				cache: &fake.MockClient{
 					MockDescribeCacheSubnetGroupsRequest: func(input *awscache.DescribeCacheSubnetGroupsInput) awscache.DescribeCacheSubnetGroupsRequest {
 						return awscache.DescribeCacheSubnetGroupsRequest{
-							Request: &aws.Request{HTTPRequest: &http.Request{}, Data: &awscache.DescribeCacheSubnetGroupsOutput{
+							Request: &aws.Request{HTTPRequest: &http.Request{}, Retryer: aws.NoOpRetryer{}, Data: &awscache.DescribeCacheSubnetGroupsOutput{
```

Comment:
This `aws.NoOpRetryer{}` is the only change in many of these test files. It is simply an artifact of bumping the SDK version. Feel free to skip over these changes.